<launch>
    <node pkg="drive_control" type="DriveControl.py" name="drive_controller" output="screen" />
</launch>
